BEP82.md

```diff
@@ -0,0 +1,72 @@
+# BEP-82: Token Owner Changes
+
+## 1. Summary
+This BEP describes the changes related to the token owner who issued a token on Binance Chain.
+
+## 2. Abstract
+Currently, many token-related transactions, such as token listing, minting, burning, etc., can only be proposed by the token owner who can not be changed once the token is issued on Binance Chain.
+
+This BEP proposes the solutions to provide more convenience and flexibility for these transactions.
+
+## 3. Status
+draft
+
+## 4. Motivation
+To reduce the transaction's dependence on the related token’s owner and make the ownership of token changeable, we introduce those solutions as following:
+- Providing an entrance to transfer the ownership of a specific token.
+- Removing the owner verification when handling some token related transactions.
+
+## 5. Specification
+### 5.1 Owner Transfer
+**OwnerTransfer** transaction can transfer ownership of a specific token to another address, and only the original owner has the permission to send this transaction.
+
+A fee (a fixed amount of BNB) will be charged on **OwnerTransfer** transaction.
+
+Parameters for OwnerTransfer Operation:
+
+|     **Field**       | **Type**    |    **Description**        |
+| ------------------- | ----------- | ------------------------  |
+| From                | string      | the address who wants to transfer the ownership to another address |
+| Symbol              | string      | the token symbol needs to be transferred ownership |
+| NewOwner            | string      | the address who is assigned to be the new owner of the token |
+
+### 5.2 Token Owner Verification Changes
+Currently, some token-related transactions are restricted to being proposed by token owners while others are not.  
+
+|    **Transaction Type**   |  **Verify Owner**   |
+| ------------------------- | ------------------- |
+| list token                | yes                 |
+| mint token                | yes                 |
+| burn token                | yes                 |
+| freeze token              | no                  |
+| set URI                   | yes                 |
+| atomic swap related       | no                  |
+| time lock                 | no                  |
+| time unlock               | no                  |
+| time relock               | no                  |
+| transfer                  | no                  |
+| cross transfer out        | no                  |
+| cross bind                | yes                 |
+| cross unbind              | yes                 |
+
+After the implementation of this BEP, any address can burn its own tokens. And the table will be like the below:
+
+|    **Transaction Type**   |  **Verify Owner**   |
+| ------------------------- | ------------------- |
+| list token                | yes                 |
+| mint token                | yes                 |
+| burn token                | no                  |
```

Comment:
There is another BEP that will `sync` binded token from BSC to BC so that rebalance the total supply of two token. If anyone on BC can burn, shall we support `sync` from BC to BSC too?  If we do, then all token on BSC need to implement burn interface.  @HaoyangLiu  @darren-liu